spec.html

```diff
@@ -11473,27 +11473,29 @@ <h1>Runtime Semantics: Evaluation</h1>
         <emu-alg>
           1. Let _array_ be ! ArrayCreate(0).
           1. Let _pad_ be the ElisionWidth of |Elision|; if |Elision| is not present, use the numeric value zero.
-          1. Perform Set(_array_, `"length"`, ToUint32(_pad_), *false*).
-          1. NOTE: The above Set cannot fail because of the nature of the object returned by ArrayCreate.
+          1. Perform ? Set(_array_, `"length"`, _pad_, *true*).
+          1. NOTE: The above Set may throw in the case of an Array literal's padding exceeds 2<sup>32</sup>-1 elements.
```

Comment:
s/exceeds/exceeding/
or better s/may throw in the case of an Array literal's/throws if the/